tests/agent_unittests/test_package_version_utils.py

```diff
@@ -26,6 +25,7 @@
     get_package_version,
     get_package_version_tuple,
 )
+from newrelic.packages import six
```

Comment:
This seems random. There are other places where we use six directly in the tests too. Not that it's a big deal, but maybe don't do this in this PR as it's not really related to PR's purpose.

Reply:
It actually is related, the `agent_unittests` are running on `cibuildwheel` in their wheel validation step now. Without `tox` setting up the virtual environment, `six` doesn't end up accessible, unsure if that's because it's normally installed or if it's available due to our `PYTHONPATH` shenanigans.